source/troubleshooting.txt

```diff
@@ -173,3 +173,144 @@ HOMEDRIVE, HOMEPATH, or USERPROFILE are blank
 
 This error might appear if the {+atlas-cli+} can't access your home
 directory.
+
+
+# Are there any known errors to add for ``atlas deployments``?
+
+.. _local-deploy-errors:
+
+Local Deployment Errors
+-----------------------
+
+Error: qemu exited unexpectedly with exit code -1, stderr: qemu-system-x86_64: Error: HV_DENIED
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The {+atlas-cli+} uses `Podman </https://podman.io/>`__ and `QEMU 
+<https://www.qemu.org/>`__. We have identified this issue on MacOS x86 
+architectures. Upgrade to the latest version of QEMU and restart Podman.
+
+Error: exit status 127
+~~~~~~~~~~~~~~~~~~~~~~
+
+This error might occur when you use the ``atlas deployments setup`` 
+command on Ubuntu Linux because of a `known issue 
+<https://bugs.launchpad.net/ubuntu/+source/libpod/+bug/2024394>`__ with 
+Podman. Downgrade Podman:
```

Comment:
Suggestion
```suggestion
Podman. Downgrade Podman to resolve this error:
```

Reply:
I updated only this session of the  doc:
```
Ubuntu Linux exit status 127
Q: When trying to use atlas deployments setup my setup fails with “Error: exit status 127” on ubuntu linux. What should I do?

A: Ubuntu is not supported in Public Preview. 
```
So that we can make sure we're calling out no support for Ubuntu. @boooczek FYI